src/main/java/io/vertx/ext/stomp/impl/Subscription.java

```diff
@@ -0,0 +1,145 @@
+package io.vertx.ext.stomp.impl;
+
+import io.vertx.ext.stomp.Frame;
+import io.vertx.ext.stomp.StompServerConnection;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Represents a subscription.
+ * This class is thread safe.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+public class Subscription {
```

Comment:
A general comment here (and it's kind of similar to the one I made before). The acks in this class don't really seem to do anything, so I'm a bit puzzled why we are tracking them at all.
In the case of a queue the idea with acks, is if the subscriber closes with unacked acks then the messages can be redelivered, but this never happens in this implementation afaict.
In the case of the topic, the acks also do nothing, and redelivery would never occur, so the acking should just be a no-op I believe.